### PR TITLE
[UI] Add Shift key for noVNC consoles

### PR DESCRIPTION
**Pull request by GabrielBrascher**
### Description

Some key combinations, such as Task Manager (ctrl+shift+esc) cannot be executed in noVNC console as the 'Shift' key is not supported.

This PR proposes to add a toggle Shift button for the noVNC console.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
@blueorangutan ui

**Comment by davidjumani** [comment]
@blueorangutan package

File: systemvm/agent/noVNC/app/ui.js [hunk 1]
```diff
@@ -1551,6 +1553,17 @@ const UI = {
         }
     },
 
+    toggleShift() {
+        const btn = document.getElementById('noVNC_toggle_shift_button');
+        if (btn.classList.contains("noVNC_selected")) {
+            UI.sendKey(KeyTable.XK_Shift_L, "ShiftLeft", false);
+            btn.classList.remove("noVNC_selected");
+        } else {
+            UI.sendKey(KeyTable.XK_Control_L, "ShiftLeft", true);
```
Comment (davidjumani) [inline systemvm/agent/noVNC/app/ui.js:1562]:
```suggestion
            UI.sendKey(KeyTable.XK_Shift_L, "ShiftLeft", true);
```

**Comment by davidjumani** [comment]
Works for key combos like ctrl+shift+esc or shift+number but not shift+alphabet since novnc prioritizes the ascii value of an alphabet sent rather than the combo of shift+alphabet

File: systemvm/agent/noVNC/app/ui.js (hunk 1 above)
Comment (GabrielBrascher) [reply systemvm/agent/noVNC/app/ui.js]: Thanks for the heads up @davidjumani! I've just updated this.

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**